python/lsst/ts/envsensors/socket_server.py

```diff
@@ -45,119 +47,85 @@ class SocketServer:
 
     valid_simulation_modes = (0, 1)
 
-    def __init__(self, port, simulation_mode=0):
-        self.port = port
+    def __init__(self, port: int, simulation_mode: int = 0) -> None:
+        self.name = "EnvSensor"
+        self.host: typing.Optional[str] = None  # always read on all interfaces
```

Comment:
I strongly suggest you use expose the host argument and use host=tcpip.LOCAL_HOST (if IPv4) to avoid listening on both IPv4 and IPv6 sockets -- which will make it difficult to use port=0 in unit tests, as we discovered yesterday. 

The reason for exposing the argument is to allow migration to IPv6 someday.

Similarly you may want to expose the `family` argument with the same default used by OneClientServer v0.2.0 and asyncio.start_server, but I don't know if it'll be needed and it definitely require ts_tcpip v0.2.0.

Reply:
Good suggestions. Done.